Compavt mode groups not working js-controller 5

```
023-03-31 09:56:18.643  - info: host.io-test compactgroup controller 1 terminated with code 1 (JS_CONTROLLER_STOPPED)
2023-03-31 09:56:18.661  - error: host.io-test Caught by controller[0]: internal/modules/cjs/loader.js:934
2023-03-31 09:56:18.661  - error: host.io-test Caught by controller[0]:   throw err;
2023-03-31 09:56:18.661  - error: host.io-test Caught by controller[0]:   ^
2023-03-31 09:56:18.661  - error: host.io-test Caught by controller[0]: Error: Cannot find module '/opt/iobroker/node_modules/iobroker.js-controller/build/compactgroupController.js'
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:931:15)
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:     at Function.Module._load (internal/modules/cjs/loader.js:774:27)
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:     at internal/main/run_main_module.js:17:47 {
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:   code: 'MODULE_NOT_FOUND',
2023-03-31 09:56:18.662  - error: host.io-test Caught by controller[0]:   requireStack: []
2023-03-31 09:56:18.663  - error: host.io-test Caught by controller[0]: }
2023-03-31 09:56:18.663  - info: host.io-test Restart compact group controller 1
2023-03-31 09:56:18.663  - debug: host.io-test Instances: ["system.adapter.admin.2"]

```

--- a/packages/controller/package.json
+++ b/packages/controller/package.json
@@ -101,7 +101,6 @@
         "conf/",
         "build/",
         "_service_iobroker.bat",
-        "src/compactgroupController.ts",
         "controller.js",
         "io-package.json",
         "iob.bat",

--- a/packages/controller/package.json
+++ b/packages/controller/package.json
@@ -101,7 +101,7 @@
         "conf/",
         "build/",
         "_service_iobroker.bat",
-        "compactgroupController.js",
+        "src/compactgroupController.ts",
         "controller.js",
         "io-package.json",
         "iob.bat",

--- a/packages/controller/src/compactgroupController.ts
+++ b/packages/controller/src/compactgroupController.ts
@@ -5,7 +5,7 @@
  *      MIT License
  *
  */
-const controller = require('./main');
+import { init } from './main';
 
 const compactGroup = parseInt(process.argv[2], 10);
 if (isNaN(compactGroup) || compactGroup < 1) {
@@ -13,4 +13,4 @@ if (isNaN(compactGroup) || compactGroup < 1) {
     process.exit();
 }
 
-controller.init(compactGroup);
+init(compactGroup);

--- a/packages/controller/src/main.ts
+++ b/packages/controller/src/main.ts
@@ -5480,8 +5480,12 @@ function stop(force?: boolean, callback?: () => void): void {
     });
 }
 
-// bootstrap
-function init(compactGroupId?: number): void {
+/**
+ * Initialize the controller
+ *
+ * @param compactGroupId the id of the compact group
+ */
+export function init(compactGroupId?: number): void {
     if (compactGroupId) {
         compactGroupController = true;
         compactGroup = compactGroupId;

